### PR TITLE
EWS: Assign test backend to user with message tester backend in v1

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo Users with message_tester backend in v1 shouldn't receive real smses.

### CODE REVIEW

File: custom/logistics/api.py [hunk 1]
```diff
@@ -331,9 +326,12 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
                 user.save()
         return user
 
+    def save_verified_number(self, user, phone_number):
+        user.save_verified_number(self.domain, phone_number, True)
```
Comment (gcapalbo) [inline custom/logistics/api.py]: do we need to do the message_tester check here too?

Comment (kkrampa) [inline custom/logistics/api.py]: No it's base class. This method is only inherited in ILSGateway module.

**Comment by gcapalbo** [comment]
@kkrampa looks good, just left one comment. also can you answer for me:

 - are these all the places that verified numbers get saved? this logic would have to be applied each time the `VerifiedNumber` gets saved
- do web users ever use the test backed?

**Comment by kkrampa** [comment]
@gcapalbo I have just checked in v1 and fortunately all web users use smsgh backend.

**Comment by gcapalbo** [comment]
@kkrampa ok thanks, and can you confirm that these are all the places that verified numbers get saved? this logic would have to be applied each time the VerifiedNumber gets saved

**Comment by kkrampa** [comment]
Yes, I've changed this in all places in migration code.

**Pull request merged**